engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -2259,8 +2259,8 @@ namespace dmGameSystem
 
     static inline dmhash_t ResolveDynamicTexturePath(GuiComponent* component, dmhash_t path_hash, char* buffer, uint32_t buffer_size)
     {
-        dmSnPrintf(buffer, buffer_size, "%s/%llu.texturec", component->m_Resource->m_Path, (unsigned long long) path_hash);
-        return dmHashString64(buffer);
+        int length = dmSnPrintf(buffer, buffer_size, "%s/%llu.texturec", component->m_Resource->m_Path, (unsigned long long) path_hash);
+        return dmHashBuffer64(buffer, length < 0 ? buffer_size - 1 : length);
```

Comment:
Same as before. Little gain here, but losing readability and adding risk of errors. (and also adding code)